contracts/modules/capital/Ramm.sol

```diff
@@ -417,6 +418,35 @@ contract Ramm is IRamm, MasterAwareV2 {
     );
   }
 
+  function getInitialObservations(
+    State memory initialState,
+    uint timestamp
+  ) public pure returns (Observation[3] memory initialObservations) {
+
+    Observation memory previousObservation;
+    uint endIdx = timestamp.divCeil(PERIOD_SIZE);
+
+    for (uint idx = endIdx - 2; idx <= endIdx; idx++) {
+      uint priceCumulativeAbove = previousObservation.priceCumulativeAbove;
+      uint priceCumulativeBelow = previousObservation.priceCumulativeBelow;
+
+      uint observationTimestamp = Math.min(timestamp, idx * PERIOD_SIZE);
+      uint observationIndex = idx % GRANULARITY;
+      uint timeElapsed = endIdx - 2 == idx ? INITIAL_PERIODS * PERIOD_SIZE : PERIOD_SIZE;
+
+      priceCumulativeAbove += 1 ether * initialState.eth * timeElapsed / initialState.nxmA;
+      priceCumulativeBelow += 1 ether * initialState.eth * timeElapsed / initialState.nxmB;
```

Comment:
Given `previousObservation.priceCumulativeAbove` and `previousObservation.priceCumulativeBelow` are both pointing to empty memory, they'll both be zero and the `+=` has no purpose here.